website/docs/api/plugin-methods/_category_.yml

```diff
@@ -1,2 +1,2 @@
 label: Plugin method references
-position: 1
+position: 2
```

Comment:
I'd like to keep the original sidebar positions, and put the new category at the bottom.

Reply:
Whatever you prefer, I did it like this to keep the alphabetical order.

Reply:
Addressd in 492615f.